pkg/validations/extendedversion.go

```diff
@@ -0,0 +1,36 @@
+package validations
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+
+	anywherev1 "github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/clients/kubernetes"
+	"github.com/aws/eks-anywhere/pkg/constants"
+	"github.com/aws/eks-anywhere/pkg/logger"
+	"github.com/aws/eks-anywhere/pkg/signature"
+	"github.com/aws/eks-anywhere/release/api/v1alpha1"
+)
+
+// ValidateExtendedK8sVersionSupport validates all the validations needed for the support of extended kubernetes support.
+func ValidateExtendedK8sVersionSupport(_ context.Context, _ *anywherev1.Cluster, bundle *v1alpha1.Bundles, _ kubernetes.Client) error {
```

Comment:
I am not using context, cluster and client field in this PR but I will need those for further extended support related validations.